pkgs/development/python-modules/meinheld/default.nix

```diff
@@ -9,6 +9,12 @@ buildPythonPackage rec {
     sha256 = "008c76937ac2117cc69e032dc69cea9f85fc605de9bac1417f447c41c16a56d6";
   };
 
+  patches = [
+    # Allow greenlet-1.0.0.
+    # See https://github.com/mopemope/meinheld/pull/123
+    ./ease-greenlet-requirement.patch
+  ];
+
```

Comment:
Please use subsituteInPlace.

Reply:
I usually prefer a patches (and from upstream as much as possible) since the patch will fail to apply after an update, but `substituteInPlace` can  hand long after they became useless.

Anyway, I changed for `substituteInPlace` and removed the patch.